lms/djangoapps/shoppingcart/models.py

```diff
@@ -1186,9 +1186,17 @@ def registration_code_used_for_enrollment(cls, course_enrollment):
         has been used during the course enrollment else Returns None.
         """
         try:
```

Comment:
Optional. Can remove this try/except since we're not using .get() any more. DoesNotExist not raised by filter().